tme/transformer.go

```diff
@@ -57,6 +57,9 @@ func transformConcept(tmeTerm Term, endpoint string) *BasicConcept {
 		AuthorityValue: identifier,
 		Aliases:        aliasList,
 	}
+	if tmeTerm.Enabled == false {
```

Comment:
If the `tmeTerm` does not contain a tag for `enabled` at all, then the flag will be set to `false`. This is because you use a `bool` and not a pointer to a `*bool`, which means `omitempty` isn't working as intended